단위 테스트 생성 및 통합 테스트 제거

ber01 opened the pull request:
- RestTemplate를 모킹하여 단위 테스트를 생성하였습니다.
- 기존의 통합 테스트는 제거하였습니다.

ber01 committed:
test: LostArkApiServiceMockTest

ber01 committed:
chore: LostArkApiServiceTest 제거

f-lab-troy commented on src/test/java/me/minkh/app/service/LostArkApiServiceMockTest.java:
```diff
@@ -0,0 +1,78 @@
+    void getProfiles() {
+        // given
+        String characterName = "성공하는_아이디";
+        String body = "{\"CharacterImage\":\"https://img.lostark.co.kr/armory/3/4e981415a43bbcec46957c00118d2759a20aff8a22aee5c817cb3b1d3b4f3280.png?v=20240110164151\",\"ExpeditionLevel\":291,\"PvpGradeName\":\"2단\",\"TownLevel\":70,\"TownName\":\"우체통\",\"Title\":null,\"GuildMemberGrade\":\"일반 길드원\",\"GuildName\":\"단합안되는쓰레기길드\",\"UsingSkillPoint\":420,\"TotalSkillPoint\":420,\"Stats\":[{\"Type\": ...
```
minor: 이와 같이 긴 string은 가독성을 위해 외부 json파일로 빼두고 테스트시 읽어들여 작업하는 방법을 추천드립니다.

ber01 replied:
07f5afaf46f83f36e18fad9403ac45f76f4cdf00

분리하였습니다.

ber01 committed:
test: profile.json 분리

test: ci 실패로 인한 출력

test: 원복해서 되는지 확인

test: 성공했을 때랑 완전히 똑같이 변경

test: injectMocks 어노테이션 추가

test: actions 캐시 날리고 재시도

refactor: MockBean 으로 변경
